packages/perseus-editor/src/widgets/free-response-editor.tsx

```diff
@@ -107,46 +108,61 @@ class FreeResponseEditor extends React.Component<Props> {
     render(): React.ReactNode {
         return (
             <View>
-                <label className={css(styles.textOptionWithLabelContainer)}>
-                    <HeadingSmall>Question</HeadingSmall>
-                    <textarea
-                        value={this.props.question}
-                        onChange={(e) =>
-                            this.props.onChange({question: e.target.value})
-                        }
-                    />
-                </label>
-                <label className={css(styles.textOptionWithLabelContainer)}>
-                    <HeadingSmall>Placeholder</HeadingSmall>
-                    <textarea
-                        value={this.props.placeholder}
-                        onChange={(e) =>
-                            this.props.onChange({placeholder: e.target.value})
-                        }
-                    />
-                </label>
-                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control -- TODO(LEMS-2871): Address a11y error */}
-                <label className={css(styles.textOptionWithLabelContainer)}>
-                    <HeadingSmall>Allow unlimited characters</HeadingSmall>
-                    <Checkbox
-                        checked={this.props.allowUnlimitedCharacters}
-                        onChange={(val) =>
-                            this.props.onChange({
-                                allowUnlimitedCharacters: val,
-                            })
+                <LabeledField
+                    label={<HeadingSmall>Question</HeadingSmall>}
+                    field={
+                        <textarea
+                            value={this.props.question}
+                            onChange={(e) =>
+                                this.props.onChange({question: e.target.value})
+                            }
+                        />
+                    }
+                    styles={{root: styles.labeledInputField}}
```

Comment:
Just out of curiosity, what's the `root` thing? Pre-coffee on a Monday morning and I feel like I haven't seen that before.

Reply:
Since the `LabeledField` component has multiple components and states, the `styles` prop accepts separate style objects for `root` (the top-level `LabeledField` element), `label` (the label element), and `error` (probably styles to apply to the `field` element when it's an an error state). As far as I know, `LabeledField` is the only Wonder Blocks component that does this.